### DISCUSSION TITLE
Get Early Access to Smart Connections `v2.1` - Share Your Experience!

### DISCUSSION THREAD

**Discussion by brianpetro** [brianpetro/obsidian-smart-connections#515] [Announcements]
Hello, Smart Connections community!

When I started using Obsidian, watching screencasts and walkthroughs of others using it was _invaluable_ to my learning process.

With the upcoming release of Smart Connections 2.1, which will introduce exciting new features, including the ability to use Anthropic and Google models in Smart Chat, I'm offering early access to version 2.1 to anyone making positive contributions to the Smart Connections community.

Here's how you can participate:

1. Record a screencast or video sharing how you use Smart Connections. 
2. Upload your video to a platform like Loom, YouTube, Vimeo, or any other hosting service (the video must be public).
3. Reply to this thread with the link to your video and email `brian@smartconnections.app`.

Once I receive your video submission, I'll send you instructions for accessing the early release of  Smart Connections 2.1 with the new Anthropic and Google model integrations in the Smart Chat.

Video ideas: how Smart Connections: 
- has enhanced your note-taking experience
- helped uncover new connections
- sparked creative ideas

Requirements:
- the video should be at least 1 minute
- the audio should be you explaining your thoughts/process

Enabling others to see how they, too, can utilize Smart Connections will be an invaluable contribution to the community.

Thank you for being part of the Smart Connections community. I look forward to watching your video submissions and seeing how you're leveraging the power of Smart Connections!

🌴

**Comment by Stofferdk** [comment]
How far is the mobile beta version?

Would love to help and test, but I am mobile only.

And this is exactly what I am searching for

**Reply by brianpetro** [reply]
Unfortunately, not far enough for testing yet.

🌴

**Comment by Stofferdk** [comment]
Ahh okay, any update on the timeline?

Hmm should be a way to just get the possible to chat with your notes on mobile, just outside of obsidian?

I guess you are already indexing notes and connecting a LLM?
That seems like the key feature and how Ai can help you uttilize our PKM.  

Like this post:
https://www.reddit.com/r/selfhosted/s/yC2SnkgGfz

Amazing work you are doing btw.

**Reply by brianpetro** [reply]
The way I use mobile is through ChatGPT and the Smart Connect - Obsidian GPT, but this also requires desktop to be running the Smart Connect software.

No ETA on the mobile right now.

🌴

**Comment by Richie-Peak** [comment]
Great Idea!

**Comment by andrewf13** [comment]
How to get access to the v2.2 Beta please?
Very interested in using it on my Android mobile

**Reply by christianngarcia** [reply]
v2.2 is in beta-testing with supporters, currently no ETA on when it will hit general release 🌼

**Comment by jroachgit** [comment]
Hey Brian, I'm using smart connections within obsidian, with the 'smart chat' function. I'm trying to use the smart connections via: smart connect -> open GPT, but then it asks me to log into connect.smartconnections.app - is that the right process, or did I miss a turn?
I can't use it unless I pay for it first by being a supporter?

**Reply by christianngarcia** [reply]
You can try it for free, see this video https://youtu.be/TDUWvOc7xJo?si=Buai2Vb7cMAPDDxF 🌼
